wiki/Tournaments/MALE/en.md

```diff
@@ -0,0 +1,281 @@
+---
+tags:
+- Highlander Con
+- LAN
+- MALE
+- Riverside
+- UCR
+---
+
+# Most Amazing Lan Ever
+
+<!-- ![MALE Logo](img/logo.png) -->
+
+The **Most Amazing Lan Ever** (***MALE***) is an osu! and osu!mania BYOC LAN tournament hosted by ![][flag_US] [ForceJansen](/users/5566923) and ![][flag_US] [Chest Flattener](/users/4292614). It will take place during *Highlander Con 2019*, a competitive gaming festival hosted by Highlander Gaming at the University of California, Riverside (*UCR*).
+
+## Event Details
+
+The Most Amazing Lan Ever will be run over the duration of Highlander Con, from roughly 10:00 a.m. to 10:00 p.m. PST on Sunday, January 27th, 2019. The schedule of matches will be determined as the event goes on.
+
+Nearest to parking lot 19, the Highlander Union Building is where most of the events for Highlander Con will be taking place. The Most Amazing Lan Ever will be in HUB 260, on the second floor. To be allowed in, you must [register for Highlander Con 2019](https://www.eventbrite.com/e/highlander-con-2019-tickets-54040467493) and have your ticket ready to scan on entry.
+
+## Prizes
+
+All prizes will be awarded to both the osu! and osu!mania winners.
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | Profile badge, Aorus K7 keyboard |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | Aorus M3 mouse |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | AMP500 mousepad |
+
+![](img/badge-osu.png) ![](img/badge-mania.png)
+
+## Organisation
+
+The Most Amazing Lan Ever is run by various osu! community members at UCR.
+
+| Position | Member(s) |
+| :-- | :-- |
+| Hosts | ![][flag_US] [ForceJansen](/users/5566923), ![][flag_US] [Chest Flattener](/users/4292614) |
+| Organizers | ![][flag_US] [ForceJansen](/users/5566923) |
```

Comment:
Do host and oraganizers have different role? also change to Organisers

Reply:
apparently not